src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailHandler.java

```diff
@@ -39,22 +40,39 @@ public MailHandler(Properties props, Authenticator auth) throws MessagingExcepti
         connectToStore();
```

Comment:
I'd just use a resource try / catch like

```java
try (connection = handler.connect()) {
    // fetch mails and read them here
} catch (IOException e) {
    // oops, something went wrong
}
```

Resource try / catch automatically closes the resource if it's autoclosable and was open.